js/components/configPanel.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Panel component for video quality selection
+ *
+ * @module VideoQualityPanel
+ */
+var React = require('react'),
+    CONSTANTS = require('../constants/constants'),
+    Utils = require('./utils'),
+    AutoplaySwitch = require('./autoplaySwitch');
+
+var ConfigPanel = React.createClass({
+    handleVideoQualityClick: function () {
+        this.props.toggleQualityAction();
+    },
+
+    render: function() {
+			var videoQualityText = Utils.getLocalizedString(this.props.language, CONSTANTS.SKIN_TEXT.VIDEO_QUALITY, this.props.localizableStrings);
+			var autoPlayText = Utils.getLocalizedString(this.props.language, CONSTANTS.SKIN_TEXT.AUTOPLAY_VIDEOS, this.props.localizableStrings);
+
+			var videoQualityItem;
+			if (this.props.controller.state.videoQualityOptions.availableBitrates) {
```

Comment:
How about `var videoQualityItem = this.getVideoQualityItem()`? It's currently hard to read.